vividus-plugin-rest-api/src/main/java/org/vividus/http/MimeTypeUtils.java

```diff
@@ -44,13 +47,28 @@ public static String getMimeTypeFromHeadersWithDefault(Header... headers)
     }
 
     public static Optional<String> getMimeTypeFromHeaders(Header... headers)
+    {
+        return getContentTypeHeader(headers).map(HeaderElement::getName);
+    }
+
+    public static Charset getCharsetFromHeaders(Header... headers)
```

Comment:
having this method it's not `MimeTypeUtils` anymore, let's rename to `ContentTypeHeaderParser`